### PR TITLE
Add Badges to README

### PR DESCRIPTION
**Pull request by paroma**
Added badges to README.

- PyPI and Conda badges remain

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,11 @@
 
 [![Build Status](https://travis-ci.org/HazyResearch/snorkel.svg?branch=master)](https://travis-ci.org/HazyResearch/snorkel)
```
Comment (vincentschen) [inline README.md]: should this point to `branch=redux` for now?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,12 @@
 <img src="figs/logo_01.png" width="150"/>
 
-[![Build Status](https://travis-ci.org/HazyResearch/snorkel.svg?branch=master)](https://travis-ci.org/HazyResearch/snorkel)
-[![Documentation](https://readthedocs.org/projects/snorkel/badge/)](http://snorkel.readthedocs.io/en/master/)
+[![Build Status](https://travis-ci.org/HazyResearch/snorkel.svg?branch=redux)](https://travis-ci.org/HazyResearch/snorkel?branch=redux)
+[![Documentation](https://readthedocs.org/projects/snorkel/badge/?version=redux)](https://snorkel.readthedocs.io/en/master/?badge=redux)
```
Comment (vincentschen) [inline README.md]: this should link to the redux branch for now, right? https://snorkel.readthedocs.io/en/redux/

**Comment by henryre** [comment]
We probably don't need to ship this just yet right? Until we merge into master. Let's not make more work for ourselves.

**Comment by bhancock8** [comment]
It's nice to see in the meantime though for general code climate, and it's an easy find and replace. I'd vote in favor.

**Comment by vincentschen** [comment]
@paroma — maybe easiest thing is branching off of master again with the changes?

**Comment by bhancock8** [comment]
Subsumed by #1400

**Pull request closed without merging**